src/main/webapp/charities.html

```diff
@@ -17,8 +18,23 @@
             <li><a href="/community.html">Community</a></li>
         </ul>
     </nav>
-    <Label id = "text" class="h1-label">Charities</Label>
-    <p>This page contains the locations of some charities across the United States. Hover over a pin to find the name of the charity!</p>
-    <div id="map"></div>
+    <p>This page allows you to search for charities across the United States and also has a map showing you charities across the country. Hover over a pin to find the name of the charity!</p>
+    <div id="search-container">
+        <textarea id="search-name" rows="1">Enter Name</textarea>
```

Comment:
For <textarea> I suggest using : placeholder = "Enter Name" like this : 
<textarea id="search-name" rows="1" placeholder = "Enter Name">Enter Name</textarea>
This way the user sees the"Enter Name" message but they don't have to delete it to type in its place. (currently, not deleting the "Enter Name" for example in one field might lead to not finding any result for type "Arts", even though the records for Arts exist)

Reply:
I have the code set up so that if there's Enter Name or Enter City its treated as empty, but I'll definitely use your idea because it's better practice. Thank you :)